workers/unity/Packages/io.improbable.gdk.debug/WorkerInspector/Codegen/VisualElementExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine.UIElements;
+
+namespace Improbable.Gdk.Debug.WorkerInspector.Codegen
+{
+    public static class VisualElementExtensions
+    {
+        public static void ShiftRightMargin(this VisualElement element, uint nest, float offset = -11)
+        {
+            var style = element.style;
+            var val = style.marginRight.value.value;
+            style.marginRight = new StyleLength(val + offset * nest);
+        }
+    }
```

Comment:
I'd possibly make this internal and expose the assembly internals to the generated code.

Probably wouldn't want this to clutter code completion for users using `VisualElement`s elsewhere.